js/src/data/utils.js

```diff
@@ -85,6 +85,18 @@ export function getReportQuery( category, type, query, dateReference ) {
 	return reportQuery;
 }
 
+function replacer( key, value ) {
```

Comment:
Can we document what is the purpose of this function?

Reply:
Sure. Added in https://github.com/woocommerce/google-listings-and-ads/pull/2544/commits/69fc1b485b03de3c2e7cf5e7162013bd8f3fb588.